docs/topics/db/models.txt

```diff
@@ -720,6 +720,17 @@ Django places some restrictions on model field names:
 
 #. A field name cannot end with an underscore, for similar reasons.
 
+#. Additionally, the word check is a reserved keyword in Django and cannot be
+   used as a field name. This is because check is a special method in Django models.
+   To work around this limitation, consider using another field name, or if you
+   need to keep check as the database column name, use the :attr:`~Field.db_column`
+   option. For example::
+
+
+       class Example(models.Model):
```

Comment:
This is now a bit redundant with the reference to `db_column` directly below this. In my opinion it's not needed.

Reply:
Thank you. I removed the additional explanations.